draft-ietf-quic-transport.md

```diff
@@ -3929,8 +3929,11 @@ max_ack_delay (0x000b):
 
 : The maximum ACK delay is an integer value indicating the
   maximum amount of time in milliseconds by which the endpoint will delay
-  sending acknowledgments.  If this value is absent, a default of 25
-  milliseconds is assumed.
+  sending acknowledgments.  This value should include the receiver's expected
+  delays in alarms firing.  For example, if a receiver specifies a 5ms
+  max_ack_delay and alarms commonly fire up to 1ms late, then it should set
+  a timer for 4ms.  If this value is absent, a default of 25 milliseconds is
```

Comment:
I would do this the other way around.  If a receiver sets ack delay timers at 5ms and alarms could fire up to 1ms late, then it should advertise a max_ack_delay of 6ms.